msg/mission_result.msg

```diff
@@ -15,3 +19,5 @@ bool flight_termination		# true if the navigator demands a flight termination fr
 bool item_do_jump_changed	# true if the number of do jumps remaining has changed
 uint16 item_changed_index	# indicate which item has changed
 uint16 item_do_jump_remaining	# set to the number of do jumps remaining for that item
+
+uint8 execution_mode	# indicates the mode in which the mission is executed
```

Comment:
Is this used?

Reply:
@dagar sorry for the delayed response. This is nowhere directly used. But I added it so that is is logged somewhere and can be used for debugging purpose.